package.json

```diff
@@ -51,10 +51,9 @@
   },
   "dependencies": {
     "@canonical/jaaslib": "0.6.0",
-    "@canonical/jujulib": "1.3.2",
+    "@canonical/jujulib": "^2.0.0-beta.1",
```

Comment:
Once Juju 2.9 gets closer to official release I'll create the actual `2.0.0` release of jujulib with the most recent schema and update this.